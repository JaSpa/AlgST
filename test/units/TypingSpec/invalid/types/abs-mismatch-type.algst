let f : () -> Int = \(i : Int) -> i in ()
