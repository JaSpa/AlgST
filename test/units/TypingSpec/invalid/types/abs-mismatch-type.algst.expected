
1:23: error:
	 Expression
	     Î»(i:Int) -> i
	 has type
	     Int -> Int
	 but context demands a subtype of
	     () -> Int